Makefile

```diff
@@ -0,0 +1,15 @@
+CC ?= gcc
```

Comment:
```suggestion
CC ?= cc
```

Removes the dependencies for Mac

Reply:
> ```suggestion
> CC ?= cc
> ```
> 
> Removes the dependency of gcc for Mac

Mac has `gcc`, its an alias to `clang`.

Reply:
so? On my machine, CC is set to `cc` already. Then we compile with `cc` on this machine.

Reply:
that's the magic of `?=`